### ISSUE TITLE
Poor yaml syntax when using stephpy/vim-yaml (and also Polyglot)

### ISSUE THREAD

**Issue by TheWall89** [dracula/vim#162]
### What happened
Syntax highlight is not colouring YAML keys and un-quoted values with different colours.

### What I expected to happen
I expect the key to have a different colour than the value.

### Screenshot
![drac_issue](https://user-images.githubusercontent.com/2626117/70984346-9ab85900-20ba-11ea-8d04-15365c60d2c4.png)

With `darkblue` theme, syntax is better:
![darkblue_ok](https://user-images.githubusercontent.com/2626117/70984373-aa37a200-20ba-11ea-8e4f-3f6959efb293.png)

### Machine Info

- **Vim type (`vim`/`gvim`/`neovim`)**: vim
- **Vim version**:  VIM - Vi IMproved 8.1 (2018 May 18, compiled Nov  7 2019 22:30:47)
- **OS**: Linux archasus 5.4.3-zen1-1-zen #1 ZEN SMP PREEMPT Fri, 13 Dec 2019 09:40:17 +0000 x86_64 GNU/Linux
- **Terminal/Terminal Emulator/VTE**: Konsole
- **`TERM` environment variable**: xterm-256color

### Additional Info

I checked out the file [yaml.vim](https://github.com/dracula/vim/blob/master/after/syntax/yaml.vim). It looks like Dracula uses matches and keyword from the default Vim syntax file for YAML. stephpy/vim-yaml uses different ones: https://github.com/stephpy/vim-yaml/blob/master/after/syntax/yaml.vim .

**Comment by dsifford** [comment]
Yaml is supported and works properly using the built-in yaml syntax.

Feel free to open a PR which adjusts your syntax highlighting to the built-in syntax.

**Comment by TheWall89** [comment]
Unfortunately I'm not good at VIM syntax/plugin development :) But I'll give it a try when I have time.
In the meantime, I found a workaround for `sheerun/vim-polyglot`. Add this to `.vimrc`:

```
let g:polyglot_disabled = ['yaml']
```
And YAML files are beatiful with Dracula:
![image](https://user-images.githubusercontent.com/2626117/71002085-e41a9f80-20de-11ea-987c-d8cef32c4d37.png)

**Comment by benknoble** [comment]
If the issue is that the other plugin defines new syntax groups, then you just need to track those down (in the syntax file you linked) and add new links in the style of `after/syntax/yaml.vim`

**Comment by dsifford** [comment]
What's the status of this? Can this be closed now?

**Comment by benknoble** [comment]
@dsifford it's unfortunate, but most of the comments re: the issue were on the PR. AFAIK, we are still waiting on https://github.com/stephpy/vim-yaml/issues/22 (which, at this point, I don't expect to see fixed).

I think, given that the workaround "works," we can close with permission of @TheWall89 

If @TheWall89 is bent on using `vim-polyglot` without this workaround, he should probably

1. fork stephpy/vim-yaml
2. Fix it
3. Submit an issue/PR with polyglot to replace stephpy's code with his by changing the build script

**Comment by TheWall89** [comment]
Hi guys. Yes, I'm closing this myself.

I'll try to issue a PR to stephpy and, if this is accepted, re-issue a new PR here to color the right groups with Dracula. Bye.

**Comment by benknoble** [comment]
@TheWall89 if stephpy won't respond, consider forking as I outlined above ^

Best of luck🧛‍♂️
